install.ps1

```diff
@@ -877,12 +718,200 @@ if (-not $noGui.IsPresent) {
+      foreach($package in $selectedPackages) {
+           $newXmlNode = $packages.AppendChild($configXml.CreateElement("package"))
+           $newXmlNode.SetAttribute("name", $package)
+       }
+    }
+
+    # Function that resets the checkboxes to match the config.xml
+    function Set-InitialPackages {
+        foreach ($checkBox in $checkboxesPackages){
+            $package =$checkbox.Text.split(":")[0]
+            if (($checkbox.Checked) -and ($package -notin $packagesToInstall)){
+                $checkBox.Checked = $false
+            }else{
+              if ((-not $checkbox.Checked ) -and ($package -in $packagesToInstall)){
+                 $checkBox.Checked = $true
+              }
+            }
+        }
+    }
+
+    # Function that checks all the checkboxes
+    function Select-AllPackages {
+        foreach ($checkBox in $checkboxesPackages){
+            $checkBox.Checked = $true
         }
+    }
+
+    # Function that unchecks all the checkboxes
+    function Clear-AllPackages {
+	foreach ($checkBox in $checkboxesPackages){
+            $checkBox.Checked = $false
+        }
+    }
+
+    # Funtion that returns an array of packages that belong to a specific category
+    function Get-PackagesByCategory{
+        param (
+         [string]$category
+        )
+        return $packagesByCategory[$category]
+    }
+
+    Add-Type -AssemblyName System.Windows.Forms
+    [System.Windows.Forms.Application]::EnableVisualStyles()
+
+    $formCategories                            = New-Object system.Windows.Forms.Form
+    $formCategories.ClientSize                 = New-Object System.Drawing.Point(1015,800)
+    $formCategories.text                       = "FLAREVM Package selection"
+    $formCategories.StartPosition              = 'CenterScreen'
+    $formCategories.TopMost                    = $true
+
+    if ([string]::IsNullOrEmpty($customConfig)) {
+        $textLabel = "Select packages to install. The default configuration (recommended for reverse engineering) is pre-selected."
+    } else {
+	$textLabel = "Select packages to install. The provided custom configuration is pre-selected."
+    }
+
+    $labelCategories                = New-Object system.Windows.Forms.Label
+    $labelCategories.text           = $textLabel
+    $labelCategories.AutoSize       = $true
+    $labelCategories.width          = 25
+    $labelCategories.height         = 10
+    $labelCategories.location       = New-Object System.Drawing.Point(30,20)
+    $labelCategories.Font           = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
+
+    $panelCategories                = New-Object system.Windows.Forms.Panel
+    $panelCategories.height         = 700
+    $panelCategories.width          = 970
+    $panelCategories.location       = New-Object System.Drawing.Point(30,30)
+    $panelCategories.AutoScroll     = $true
+
+    $resetButton                 = New-Object system.Windows.Forms.Button
+    $resetButton.text            = "Reset"
```

Comment:
[nitpick] I think renaming this button to `Rest to default config` or `Reset to custom config` (depending if a custom config is provided) would make it easier to understand what _reset_ means.

Reply:
I believe the size of the reset would be too large, not sure if that will look good. How about adding it to the label, for example: Select packages to install. The provided custom configuration is pre-selected. Click on the reset button to restore the configuration.

Reply:
This is a good idea! :+1: